### PR TITLE
Build errors in master 

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Fixes build errors in master. Mostly indents in procedures, but one page in the Upgrade Guide had some terrible syntax as well.

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

File: modules/upgrade/pages/server-major-upgrade-uyuni.adoc [hunk 1]
```diff
@@ -12,57 +12,46 @@ For important extra information about your upgrade, see the release notes at htt
...
+.Procedure: Migrating the {productname} Server
 . Before running the [command]``server-migrator.sh`` script, check whether the most recent version of the [package]``susemanager`` package is installed:
 +
 ----
 zypper ref
 zypper up susemanager
 ----
-
 . Run the [command]``/usr/lib/susemanager/bin/server-migrator.sh`` script to upgrade the base OS and {productname} Server.
-// During migration the script will stop the spacewalk services.
```
Comment (Loquacity) [inline modules/upgrade/pages/server-major-upgrade-uyuni.adoc:29]: This comment was breaking the procedure:

image

File: modules/upgrade/pages/server-major-upgrade-uyuni.adoc [hunk 2]
```diff
@@ -12,57 +12,46 @@ For important extra information about your upgrade, see the release notes at htt
...
-Postgresql 12 or postgresql 13?
-
-This depends on the underlying operating system.
-Leap 15.2 shipped version 12 and this is the version used by uyuni on that OS.
-Leap 15.3 is expected to have postgresql 13, so expect postgresql version 13.
-
-====
-
-====
-
```
Comment (Loquacity) [inline modules/upgrade/pages/server-major-upgrade-uyuni.adoc:58]: This admon was breaking the procedure:

image

**Review by keichwa — COMMENTED** [review]
thanks a million!

**Review by keichwa — APPROVED** [review]

**Review by jcayouette — APPROVED** [review]
Looks good thanks!

**Pull request merged**